### ISSUE TITLE
API: Update Allowance Holdings Account Number error message 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#1365]
### Definition of Ready
- [x] Does this meet the INVEST criteria?
- [x] Does this have a clearly defined user?
- [ ] Does this have an estimate?
- [x] Does this have acceptance criteria?
- [x] Does this have sufficient context (wireframes, business rules, etc.)?
- [x] Is this traceable to a feature, epic, or PI?

## User Story
As a CAMPD API user  I want to receive an error message for account number so that I can fix my query. 

## Acceptance Criteria
Given that a user has entered one or more invalid values for an account number 
When the API endpoint is executed
Then the system displays error message: FV-100 (see below)

## Context
[Error message repo ](https://teams.microsoft.com/l/file/E95CEE6E-3EC6-4619-A8C9-CEEA00ECD464?tenantId=88b378b3-6748-4867-acf9-76aacbeca6a7&fileType=xlsx&objectUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam%2FShared%20Documents%2FGeneral%2FAnalysis%2C%20User%20Research%20and%20Design%2FAnalysis%2FGeneral%20Overview%2FError%20Messages-EPA%20CAMD-v2.1.xlsx&baseUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam&serviceName=teams&threadId=19:6f05dfa026c84672a2cc1bfa72c58f14@thread.skype&groupId=21a5e1cd-d3f1-48a6-801f-0d34e2d63d23)

Account Number:
1. Validate that the account number is 12 alphanumeric characters long

Error Message: Error Message: One or more < insert property > are not valid. Refer to the list of available < insert property > for valid values [ placeholder for link to endpoint ] FV-100

## Testing
Test Case: [TC179](https://cvpcorp.testquality.com/project/11428/plan/20060/test/283516)
Tester: @JanellC 

### Definition of Done
- [x] Have all Acceptance Criteria been agreed to and validated by the scrum team?
- [x] Has the code been unit tested and peer reviewed?
- [x] Have the functional tests been executed?
- [x] Have all defects been reviewed and dispositioned, resolved, or deferred?
- [ ] Has the user story been accepted by the Product Owner?
- [x] Is the user story ready to be deployed to the test/staging environments?

**Comment by sdhawan16** [comment]
Completed development work, will create PR once 1348 is merged

**Comment by JanellC** [comment]
TC179 passed @tlageman-epa 


https://easey-dev.app.cloud.gov/api/account-mgmt/allowances/holdings?accountNumber=000000000001|%200000000000011
